test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -107,6 +107,13 @@ public void Basic_Runtime()
         }
 
         [Fact]
+        public void Sections_Runtime()
```

Comment:
A lot of other things wont actually work at runtime, having this is probably misleading. Heck, I probably wouldn't have any of the runtime tests since they'll just be noise.